infra/global-metrics/init-letsencrypt.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
```

Comment:
This is a duplicate from ursa/ folder

Reply:
Can pass create cars in place of urls for variability

Reply:
Its actually slightly different and doesn't run any prompts, because its used in the cloud config in terraform. I could maybe refactor for a single script that has a promptless option?

Reply:
i just diffed the two from `/ursa` and this folder. They are exactly the same, aside from the domain name

Reply:
the original script was intended to handle the `DOMAINS` var for any use case.